deps-repository/pom.xml

```diff
@@ -0,0 +1,33 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
So, I did some changes to incorporate your sugestions of using ${env_var:M2_REPO} , however I just realized we can also removed the `deps-repository` entirely by just using `repository/` directly. I think this is best since it removes all duplication. For example if you add another dependency like gson, or change it's version, we need to change not just the POM, but also the `ide.target` file.

With `${project_loc:eclipse-language-service}/deps-repository/`... there is no such need, and also no deps-repository duplication. Yes the target platform will contain eclipse-language-service bundle as well, (which is what I was originally triyng to avoid), but I just remembered, the workspace plugins override the target plaftform ones, so it should not be a problem I think.

Also, I saw your comment in in m2e about using https://github.com/andriusvelykis/pde-target-maven-plugin . It looks quite interesting, but I haven't been able to test it yet: my Tycho builds are taking ages to build because of downloading Eclipse bundles (it seems the servers are very slow). Will come back once I've been able to try it and test it